src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -535,42 +518,60 @@ public with sharing class CRLP_RollupUI_SVC {
             // CMT data will generate a unique RecordName to use for the new record.
             CRLP_RollupCMT.Rollup rollupCMTObj = (CRLP_RollupCMT.Rollup) JSON.deserialize(rollupCMT, CRLP_RollupCMT.Rollup.class);
 
-            // TODO Remove this code after the UI properly sets the fields. For need it to test inserting new rollups
```

Comment:
After removing this, I'm getting errors for the amountObject missing. We can put this back, set the amount object directly in the UI, or set it in Apex, but it's not setting correctly just yet.